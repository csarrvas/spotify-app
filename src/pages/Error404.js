import React from 'react';
import './styles/error404.scss';
import './styles/error404.scss';

const Error404 = () => {
  return (
    <div id="error404">
      <img alt="error 404" src="/images/error-404.png" />
      <p>Page not found</p>
    </div>
  );
}
 
export default Error404;