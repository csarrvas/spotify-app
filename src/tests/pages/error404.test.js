import React from 'react';
import { shallow } from 'enzyme';
import Error404 from '../../pages/Error404';

test('should render Error404 correctly', () => {
  const wrapper = shallow(<Error404/>);
  expect(wrapper).toMatchSnapshot();
});