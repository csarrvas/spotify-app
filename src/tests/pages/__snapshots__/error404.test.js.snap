// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Error404 correctly 1`] = `
<div
  id="error404"
>
  <img
    alt="error 404"
    src="/images/error-404.png"
  />
  <p>
    Page not found
  </p>
</div>
`;
